scripts/generate-schemas.py

```diff
@@ -50,6 +50,7 @@ def empty_schema(schema_name):
         "type": "object",
         "additionalProperties": False,
         "properties": {},
+        "anyOf": [],
```

Comment:
We probably don't need to add the key here since we're just going to remove it if it's empty